### PR TITLE
Staging: ppe-monitoring-and-teacher-dashboard

### PR DESCRIPTION
**Pull request by osman-goni-cse**

### CODE REVIEW

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 1]
```diff
@@ -724,6 +724,30 @@ const TITLE_LOCALES_BN = {
...
+    'en' : 'School Inspection Completed',
+    'bn' : 'বিদ্যালয় পরিদর্শন হয়েছে'
+  },
+  'admin.dashboard.ppe.monitoring.school.not.visited.count' : {
+    'en' : 'School Inspection Pending',
+    'bn' : 'বিদ্যালয়ের পরিদর্শন বাকি'
+  },
+  'admin.dashboard.ppe.monitoring.school.status' : {
+    'en' : 'Statistics Based on Status of Schools',
+    'bn' : 'বিদ্যালয়সমূহের স্ট্যাটাস ভিত্তিক পরিসংখ্যান'
```
Comment (NaimulHasanFahim) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:737]:
```suggestion
    'en' : 'Pre-Primary Classroom Status Statistics',
    'bn' : 'প্রাক-প্রাথমিক শ্রেণীকক্ষের স্ট্যাটাস ভিত্তিক পরিসংখ্যান'
```

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 2]
```diff
@@ -724,6 +724,30 @@ const TITLE_LOCALES_BN = {
...
+  },
+  'admin.dashboard.ppe.teacher.gender.statistics' : {
+    'en' : 'Pre-Primary Teacher Gender Statistics',
+    'bn' : 'প্রাক-প্রাথমিক শিক্ষকের জেন্ডার পরিসংখ্যান'
```
Comment (NaimulHasanFahim) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:741]:
```suggestion
    'bn' : 'প্রাক-প্রাথমিক শিক্ষকের জেন্ডারের পরিসংখ্যান'
```

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js [hunk 3]
```diff
@@ -724,6 +724,30 @@ const TITLE_LOCALES_BN = {
...
+  },
+  'admin.dashboard.ppe.teacher.training.statistics' : {
+    'en' : 'Pre-Primary Teacher Training Statistics',
+    'bn' : 'প্রাক-প্রাথমিক শিক্ষকের প্রশিক্ষণ পরিসংখ্যান'
```
Comment (NaimulHasanFahim) [inline frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js:745]:
```suggestion
    'bn' : 'প্রাক-প্রাথমিক শিক্ষকের প্রশিক্ষণের পরিসংখ্যান'
```

**Review by NaimulHasanFahim — CHANGES_REQUESTED** [review]
LGTM. Lefts some comments

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 1 above)
Comment (osman-goni-cse) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done.

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 2 above)
Comment (osman-goni-cse) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done.

File: frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js (hunk 3 above)
Comment (osman-goni-cse) [reply frontend/src/metabase/visualizations/lib/settings/dashcard_locales.js]: done.

**Review by NaimulHasanFahim — APPROVED** [review]
LGTM

**Pull request merged**